modules/1_opencv/src/opencv_coins.cpp

```diff
@@ -11,4 +14,25 @@ unsigned countCoins(const Mat& img) {
 
     // TODO: implement an algorithm from https://docs.opencv.org/master/d3/db4/tutorial_py_watershed.html
     CV_Error(Error::StsNotImplemented, "countCoins");
+    dilate(thresh, thresh,Mat::ones(3, 3, CV_8U), Point(-1, -1), 3);
+    distanceTransform(thresh, thresh, DIST_L2, 5);
+    double dist_transform;
+    minMaxLoc(thresh, &dist_transform);
+    threshold(thresh,thresh, 0.7*dist_transform,255.0, THRESH_BINARY);
+    thresh.convertTo(thresh, CV_8U);
+    std::vector<std::vector<cv::Point> > contours;
+    cv::findContours(thresh, contours, RETR_EXTERNAL, CHAIN_APPROX_SIMPLE);
+    int ncomp = contours.size();
+    int sum = 0;
+    double radius;
+    for (int i = 0; i < ncomp; ++i) {
+        radius = contourArea(contours[i]) / (2 * 3.14);
```

Comment:
Is that really radius? Can you please remind circle radius formula?

Reply:
s=pi*r^2

Reply:
So why we divide by 2?

Reply:
I fixed this mistake, but the tests failed

Reply:
Please check https://github.com/dkurt/openvino_practice/pull/85/files - the changes are the same